workday/12.1/modules/ROOT/pages/workday-connector-studio.adoc

```diff
@@ -37,34 +37,38 @@ Add Workday Connector to your Mule project to automatically populate the XML cod
 Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
 
 [[configure-input-source]]
-== Configure an Input Source
+== Configure a Source
 
-An input source initiates a flow when a specified condition is met.
+A source (listener) initiates a flow when a specified condition is met.
 You can configure one of these input sources to use with Workday Connector:
 
-* On Modified Objects +
-Starts your app when an object is created, modified, or deleted in Workday.
+* On New Objects +
+Start your app when an object is created in Workday.
 * HTTP Listener +
 Starts your app using a browser or a command such as `curl`.
 * Scheduler +
 Starts your app at specified timed intervals.
 
-To configure an *On Modified Objects* input source:
+To configure an *On New Objects* input source:
 
-. In the *Mule Palette* view, select *Workday > On Modified Objects*.
-. Drag *On Modified Objects* to the Studio canvas.
-. On the *On Modified Objects* configuration screen, optionally change the value of the *Display Name* field.
+. In the *Mule Palette* view, select *Workday > On New Objects*.
+. Drag *On New Objects* to the Studio canvas.
```

Comment:
```suggestion
. Drag *On New Objects* source to the Studio canvas.
```

Reply:
You can omit this, since we currently we do not do it with other operations/sources...but I felt it was missing something.